ac.soton.eventb.emf.containment.releng/pom.xml

```diff
@@ -18,45 +18,38 @@
   <!-- ID of this artifact -->
   <artifactId>releng</artifactId>
   <!-- REQUIRED: Version number (using semantic versioning) -->
-  <version>0.0.1.release</version>
+  <version>0.0.3.release</version>
```

Comment:
do we need to skip 0.0.2? (I don't think the version 0.0.2 was ever merged so is still part of this pull request?)